### PR TITLE
Feature/people page 2

### PR DESCRIPTION
**Pull request by subinasr**

### CODE REVIEW

File: apps/main/models.py [hunk 1]
```diff
@@ -17,6 +17,12 @@ class PageType(enum.Enum):
 
     content = models.TextField()
 
+    group_picture = models.ImageField(
+        upload_to="group_picture",
```
Comment (barshathakuri) [inline apps/main/models.py:21]: Inconsistent spacing

File: config/settings.py [hunk 2]
```diff
@@ -69,7 +69,11 @@
                 "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
-            ]
+            ],
+
```
Comment (barshathakuri) [inline config/settings.py:73]: Remove blank line

File: templates/studio/about.html [hunk 3]
```diff
@@ -11,9 +11,18 @@
...
 <main class="studio">
-    <div class="navbar">
-        {% include "partials/studio_nav.html"%}
-        {% include "partials/nav.html" %}
+    <button
+        href="javascript:void(0);"
+        class="hamburger"
+        onclick="clickFunction()"
+    >
+        <ion-icon name="menu"></ion-icon>
```
Comment (barshathakuri) [inline templates/studio/about.html:19]: move Close tags in next line

File: templates/works/list.html [hunk 4]
```diff
@@ -11,22 +12,52 @@
...
+    <button
+        href="javascript:void(0);"
+        class="hamburger"
+        onclick="clickFunction()"
+    >
+        <ion-icon name="menu"></ion-icon>
+    </button>
+    <div id="myNavbar" class="navbar">
+        <a class="logo" href="{% url 'home' %}">
+            <img src={% static 'logo.png' %}>
```
Comment (barshathakuri) [inline templates/works/list.html:24]: Add close tag in img

**Review by barshathakuri — CHANGES_REQUESTED** [review]

File: apps/main/models.py (hunk 1 above)
Comment (subinasr) [reply apps/main/models.py:21]: Turns out that's how functions are created in django. Felt really new to me too.

File: templates/studio/about.html (hunk 3 above)
Comment (subinasr) [reply templates/studio/about.html:19]: Tags with no children can be left like this, per my opinion.

File: templates/works/list.html (hunk 4 above)
Comment (subinasr) [reply templates/works/list.html:24]: These are self closing tags.

**Pull request closed without merging**